### PR TITLE
Start batch testing.

### PR DESCRIPTION
**Pull request by spxtr**
Tested it out manually.

---
This change is [Reviewable](https://reviewable.kubernetes.io/reviews/kubernetes/test-infra/1185)

### CODE REVIEW

File: prow/cmd/line/main.go [hunk 1]
```diff
@@ -284,8 +287,12 @@ func (c *testClient) TestPRJenkins() error {
 		return err
 	}
 
-	resultURL := c.guberURL(strconv.Itoa(result.Number))
+	var resultURL string
+	if c.Report {
+		resultURL = c.guberURL(strconv.Itoa(result.Number))
+	}
 	c.tryCreateStatus(github.StatusPending, "Build started.", resultURL)
+	time.Sleep(2 * time.Minute)
```
Comment (rmmh) [inline prow/cmd/line/main.go]: debug?

File: prow/Makefile [hunk 2]
```diff
@@ -15,10 +15,10 @@
...
 
 
-HOOK_VERSION   = 0.54
-LINE_VERSION   = 0.27
+HOOK_VERSION   = 0.55
+LINE_VERSION   = 0.28
 SINKER_VERSION = 0.4
-DECK_VERSION   = 0.5
+DECK_VERSION   = 0.6
 SPLICE_VERSION   = 0.0
```
Comment (rmmh) [inline prow/Makefile]: shouldn't this get bumped too?

Comment (spxtr) [inline prow/Makefile]: Nah, I never deployed it in the first place.

**Comment by spxtr** [comment]
Also snuck on a commit to make our webhooks happier.

**Comment by spxtr** [comment]
@k8s-bot test this

**Pull request merged**